tests/test_on_data/test_behavior_interfaces.py

```diff
@@ -37,6 +37,10 @@ class TestFicTracDataInterface(DataInterfaceTestMixin, unittest.TestCase):
 
     save_directory = OUTPUT_PATH
 
+    def check_extracted_metadata(self, metadata: dict):
```

Comment:
Actually, before merge, can we get a test for the new starting time functionality? Or is the example data irregular?